src/frontend/App.js

```diff
@@ -51,7 +52,9 @@ const App = () => (
       <PermissionsProvider>
         <AppLoading>
           <AppProvider>
-            <AppContainerWrapper />
+            <BottomSheetModalProvider>
```

Comment:
Can you add a comment explaining why this is here.

Reply:
addressed in https://github.com/digidem/mapeo-mobile/pull/1083/commits/db02e2d8559dba0a8f138c23ff99d4d7ae0b1755